src/language-css/parser-postcss.js

```diff
@@ -32,6 +32,18 @@ function parseValueNodes(nodes) {
   for (let i = 0; i < nodes.length; ++i) {
     const node = nodes[i];
 
+    if (
+      node.type === "number" &&
+      node.unit === ".." &&
+      node.value[node.value.length - 1] === "."
```

Comment:
Will be great to add `isScssSyntax()` check here, because we don't want to parse `...` for less and css

Reply:
Can't find `isScssSyntax` anywhere in the codebase, do you mean checking options.parser ? 
If so, I don't have access to it in `parseValueNodes`

Reply:
@evilebottnawi please advise, I had another look but I can't find any way to determine whether or not I'm in scss syntax this far into the callstack

Reply:
Just pass it to `parseNestedCSS` and other nested functions

Reply:
Done!